pyfr/writers/vtk.py

```diff
@@ -475,25 +475,25 @@ def _get_array_attrs(self, sk=None):
         else:
             return names, types, comps
 
-    @memoize
```

Comment:
Why are we reordering things here?

Reply:
As I pulled them out to a mixin and put them back, when I put them back I made them alphabetical

Reply:
Revert, just adds noise when someone does a `git blame` or bisects the file.  Reordering is fine if we're making other changes to that piece of code, but otherwise is best avoided.